scripts/daily_upload_s3.coffee

```diff
@@ -19,7 +19,12 @@ db = mongojs(process.env.MONGOHQ_URL, ['articles'])
 # Setup CSV
 filename = "export_" + moment().format('YYYYMMDDhhmmss') + ".csv"
 csvStream = csv.createWriteStream { headers: true }
-writableStream = fs.createWriteStream('scripts/tmp/' + filename)
+dir = 'scripts/tmp/'
+if !fs.existsSync dir
+  fs.mkdir dir
```

Comment:
`fs.mkdir` requires a callback, and as it stands you'll like have a race condition where the script might try to write to a non-existent directory b/c `mkdir` didn't finish. You could use `fs.mkdirSync` to avoid this or better yet you can just check in the tmp folder by creating the directory and adding an empty .gitkeep file inside of it.